zoo/services/tasks.py

```diff
@@ -34,6 +34,41 @@ def fetch_sentry_project_issues(project_slug: str) -> dict:
     return result
 
 
+def fetch_sonarqube_project_links(project_key: str) -> list:
+    api_request_url = "{0}/api/project_links/search?projectKey={1}".format(
```

Comment:
I can imagine this API calls randomly failing some time. Timeouts or just some temporary hiccup. I think the calls should be wrapped using `retry`